src/keyValueStore.js

```diff
@@ -238,7 +202,7 @@ class KeyValueStore {
    *
    * @return    {Array<Object>}     Array of ordered log entry objects
    */
-  get log () {
+  async log () {
```

Comment:
why async now?

Reply:
Because the decrypt function in the IdentityWallet has to be async. If this function was sync then the api would be different for private data.